### PR TITLE
Docs8459/update guides

### PR DESCRIPTION
**Pull request by buraizu**
This PR:
- Adds new guides with example Terraform resources for which there are **not** any examples currently
- Adds a new resource to the existing Monitors guide

### CODE REVIEW

File: docs/guides/logs.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+subcategory: ""
+page_title: "Log Resource Examples"
+description: |-
+    Log Resource Examples
+---
+
+### Log Resource Examples
```
Comment (brett0000FF) [inline docs/guides/logs.md:8]: Is it correct to start these guides with an h3? I'm unsure if there is specific formatting that requires this so flagging.

File: docs/guides/metrics.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+subcategory: ""
+page_title: "Metric Resource Examples"
+description: |-
+    Metric Resource Examples
+---
+
+### Metric Resource Examples
+
+This page lists examples of how to create different Datadog monitor types within Terraform. This list is not exhaustive and will be updated over time to provide more examples.
```
Comment (brett0000FF) [inline docs/guides/metrics.md:10]:
```suggestion
This page lists examples of how to create different Datadog metric types within Terraform. This list is not exhaustive and will be updated over time to provide more examples.
```

File: docs/guides/security.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+---
+subcategory: ""
+page_title: "Security Resource Examples"
+description: |-
+    Security Resource Examples
+---
+
+### Security resource examples
+
+This page lists examples of how to create Datadog security monitoring resources within Terraform. This list is not exhaustive and will be updated over time to provide more examples.
```
Comment (brett0000FF) [inline docs/guides/security.md:10]:
```suggestion
This page lists examples of how to create Datadog Security Monitoring resources within Terraform. This list is not exhaustive and will be updated over time to provide more examples.
```

**Review by brett0000FF — DISMISSED** [review]
Just a few minor comments!

File: docs/guides/logs.md (hunk 1 above)
Comment (buraizu) [reply docs/guides/logs.md:8]: Thanks for flagging that @brett0000FF, I'm just following the format of the only existing guide:
https://github.com/DataDog/terraform-provider-datadog/blob/master/docs/guides/monitors.md?plain=1

Whether it's correct might be a matter of opinion, but my goal for now is just to maintain consistency. An update to the heading structure might be a good follow up task, so I created DOCS-9226.

**Review by brett0000FF — APPROVED** [review]

**Pull request closed without merging**